core/src/test/java/org/springframework/security/scheduling/CurrentDelegatingSecurityContextTaskSchedulerTests.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2002-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.scheduling;
+
+import org.junit.Before;
+
+import org.springframework.security.core.context.SecurityContext;
+
+/**
+ * Tests using the current {@link SecurityContext} on
+ * {@link DelegatingSecurityContextTaskScheduler}
+ *
+ * @author Giacomo Baso
+ * @since 5.5
+ *
+ */
+public class CurrentDelegatingSecurityContextTaskSchedulerTests
+		extends AbstractDelegatingSecurityContextTaskSchedulerTests {
+
+	@Before
+	public void setUp() throws Exception {
+		super.currentSecurityContextPowermockSetup();
```

Comment:
While I realize it was done this way in the past, I don't think that mocking the `create` methods for `DelegatingSecurityContextCallable` and `DelegatingSecurityContextRunnable` is necessary.

Reply:
See https://github.com/spring-projects/spring-security/pull/9532#discussion_r608824828